tools/flaky-test-reporter/run.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// run.go controls how to run functions that needs dryrun support
+
+package main
+
+import (
+	"log"
+)
+
+func run(message string, call func() error, dryrun *bool) error {
```

Comment:
any reason dryrun is a pointer and not a bool?

Reply:
it's for consistency, as it's a pointer in the beginning

Reply:
since its a flag, it will return the pointer. But, this function can take in a bool instead as we can catch this as compile time instead.

Reply:
done